### ISSUE TITLE
internal error, uncaught exception: Stack overflow

### ISSUE THREAD

**Issue by talex5** [lpw25/prof_alloc#11]
I tried following the instructions at https://blogs.janestreet.com/a-brief-trip-through-spacetime/ but I get this when I try to look at the profile:

```
$ prof_spacetime serve spacetime-88567 
Processing series...prof-spacetime: internal error, uncaught exception:
                Stack overflow
```

The trace is from running DataKitCI for a few seconds (179 MB file).

(not using `-e` as I'm on OS X, due to #4)

**Comment by talex5** [comment]
Couldn't work out how to increase the stack size on OS X, but I was able to do the processing in a Linux VM with `ulimit -s 81920` (default stack size was 8192 for OS X and Linux).

**Comment by talex5** [comment]
Solution:

    opam pin add --dev spacetime_lib

**Comment by lpw25** [comment]
Hmm, I guess I should do a release

**Comment by lpw25** [comment]
This is now released in 0.2.0
